### DISCUSSION TITLE
Connecting RFID reader to SmartEVSE v2

### DISCUSSION THREAD

**Discussion by bostjanlaba** [SmartEVSE/SmartEVSE-2#51] [Q&A] [answered]
I have found a reader on Stegen site, but it says for v3. I know v2 supports rfid but how do I connect it?

Just asking before I'd order the reader and it wouldn't fit...

Thanks

Bostjan

**Comment by mstegen** [comment, marked as answer]
Yes you can also use it on SmartEVSE v2.

YELLOW wire to the 12V terminal.
GREY wire to the GND terminal.
WHITE wire to the SW terminal.

Make sure the firmware on your controller is at least v2.20
